src/Formats/FormatFactory.cpp

```diff
@@ -259,7 +260,7 @@ InputFormatPtr FormatFactory::getInputFormat(
 
 
 OutputFormatPtr FormatFactory::getOutputFormat(
-    const String & name, WriteBuffer & buf, const Block & sample, const Context & context, WriteCallback callback) const
+    const String & name, WriteBuffer & buf, const Block & sample, const Context & context, WriteCallback callback, const bool allow_multiple_rows_in_unframed_formats) const
```

Comment:
What is `unframed format`?

Reply:
Message framing - is a way to split a stream of data into chunks (the common ways to do framing is length-prefixes or delimiters).

So unfeamed format is format w/o framing (when you can't understand when messages starts / stops). For example RowBLOB also has no framing. 

Ideas for alternative name?